server/server.go

```diff
@@ -95,6 +95,8 @@ const (
 	// terraformPluginCacheDir is the name of the dir inside our data dir
 	// where we tell terraform to cache plugins and modules.
 	TerraformPluginCacheDirName = "plugin-cache"
+
+	UseChecksApi = true
```

Comment:
What's the plan here? is this going to be a flag? Can we add a comment explaining that?